netmiko/f5/f5_tmsh_ssh.py

```diff
@@ -25,3 +25,15 @@ def tmsh_mode(self, delay_factor=1):
         time.sleep(1 * delay_factor)
         self.clear_buffer()
         return None
+
+    def check_config_mode(self, check_string=") #", pattern=""):
+        """Checks if the device is in configuration mode or not."""
+        return super().check_config_mode(check_string=check_string, pattern=pattern)
```

Comment:
I think I am going to change this to just `return True` unless you see an issue with that? That is what I typically do for platforms that have no separate config mode.

Everything else looks good on this.

Reply:
Ok no problem !

Reply:
Ok I've tried with `return True` in place of  `return super().check_config_mode(check_string=check_string, pattern=pattern)`

and the test_netmiko_config.py drops me the error below : 

```
    def test_exit_config_mode(net_connect, commands, expected_responses):
        """
        Test exit config mode
        """
        net_connect.exit_config_mode()
>       assert net_connect.check_config_mode() is False
E       assert True is False
E        +  where True = <bound method F5TmshSSH.check_config_mode of <netmiko.f5.f5_tmsh_ssh.F5TmshSSH object at 0x7fb26f50aef0>>()
E        +    where <bound method F5TmshSSH.check_config_mode of <netmiko.f5.f5_tmsh_ssh.F5TmshSSH object at 0x7fb26f50aef0>> = <netmiko.f5.f5_tmsh_ssh.F5TmshSSH object at 0x7fb26f50aef0>.check_config_mode

test_netmiko_config.py:42: AssertionError
================================================================================================ short test summary info ================================================================================================
FAILED test_netmiko_config.py::test_exit_config_mode - assert True is False
============================================================================================= 1 failed, 8 passed in 12.28s ==============================================================================================
```

Reply:
I'll add probably later commit stuff because when you modify some config and you have F5 loadbalancer in cluster, 

the prompt is as below before the change : 
`trasmontinho@(f5a)(cfg-sync Standalone)(/S1-green-P::Active)(/Common)(tmos)#`
or 
`trasmontinho@(f5b)(cfg-sync Standalone)(/S1-green-P::Standby)(/Common)(tmos)#`

As soon as you config some data the prompt change to  : 
`trasmontinho@(f5b)(cfg-sync Changes Pending)(/S1-green-P::Standby)(/Common)(tmos)#`
or 
`trasmontinho@(f5a)(cfg-sync Changes Pending)(/S1-green-P::Standby)(/Common)(tmos)#`

Reply:
Okay, that test error is fine. It is a problem with the test which I will correct.

Reply:
Ok maybe I'll drop you an issue on it. 
I've got to fill the` enable_prompt` to pass the test. If I don't, I've got a `KeyError 'enable_prompt'`
Maybe such stuff should be warn you on  missing tests if those elements are not present.

Reply:
Yeah, just put that as the same as the standard prompt on  your device so whatever the full prompt is after logging into `tmos` for the device you are testing.

I fixed the `exit_config_mode` test issue in new Pull-Request (#2153)